lib/charms/mongodb/v1/mongodb_provider.py

```diff
@@ -31,17 +31,15 @@
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 10
+LIBPATCH = 14
```

Comment:
This will be 12 not 14 soon because of https://github.com/canonical/mongodb-operator/pull/480.
I also bring this change in https://github.com/canonical/mongodb-k8s-operator/pull/328, so whoever merges first, the other will have to rebase I guess :)